server/.flaskenv

```diff
@@ -0,0 +1 @@
+FLASK_APP=server/app.py
```

Comment:
This can just go in a `.env` file - we've got python-dotenv loaded so environment variables set there will automatically be set